version.h

```diff
@@ -55,11 +55,13 @@
 #endif
 
 #if RUBY_PATCHLEVEL == -1
-#define RUBY_PATCHLEVEL_STR "dev"
+# ifndef RUBY_PATCHLEVEL_STR
```

Comment:
```suggestion
# ifdef RUBY_PATCHLEVEL_NAME
#  define RUBY_PATCHLEVEL_STR STRINGIZE(RUBY_PATCHLEVEL_NAME)
# else
```

Then `make DEFS=-DRUBY_PATCHLEVEL_NAME=shopify`.
`DEFS` is defaulted to undefined, while `cppflags` may be set.

Reply:
Thanks for the suggestion 👍